### PR TITLE
Fix doc link to Entity Framework instrumentation package

### PR DESCRIPTION
**Pull request by nilsandrey**
The link to the Entity Framework instrumentation package was pointing to the root of the Nuget packages pages instead of the specific Open Telemetry package.

### CODE REVIEW

**Review by cijothomas — APPROVED** [review]

File: content/en/docs/languages/net/automatic/instrumentations.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ stable semantic convention.
...
 | `ELASTICTRANSPORT`    | [Elastic.Transport](https://www.nuget.org/packages/Elastic.Transport)                                                                                                                                              | ≥0.4.16            | source               | [Experimental](/docs/specs/otel/versioning-and-stability) |
-| `ENTITYFRAMEWORKCORE` | [Microsoft.EntityFrameworkCore](https://www.nuget.org/packages/) **Not supported on .NET Framework**                                                                                                               | ≥6.0.12            | source               | [Experimental](/docs/specs/otel/versioning-and-stability) |
+| `ENTITYFRAMEWORKCORE` | [Microsoft.EntityFrameworkCore]([https://www.nuget.org/packages/](https://www.nuget.org/packages/OpenTelemetry.Instrumentation.EntityFrameworkCore)) **Not supported on .NET Framework**                                                                                                               | ≥6.0.12            | source               | [Experimental](/docs/specs/otel/versioning-and-stability) |
```
Comment (pjanotti) [inline content/en/docs/languages/net/automatic/instrumentations.md:51]:
```suggestion
| `ENTITYFRAMEWORKCORE` | [Microsoft.EntityFrameworkCore](https://www.nuget.org/packages/OpenTelemetry.Instrumentation.EntityFrameworkCore) **Not supported on .NET Framework**                                                                                                               | ≥6.0.12            | source               | [Experimental](/docs/specs/otel/versioning-and-stability) |
```

**Review by pjanotti — COMMENTED** [review]
A minor suggestion to keep the link like the others.

**Review by pjanotti — APPROVED** [review]
Thanks @nilsandrey!

**Comment by cartermp** [comment]
@nilsandrey Sorry for the churn here -- we recently moved this file to be under the `zero-code` folder, so there is a merge conflict.

If you'd like to resubmit, we'll gladly approve it and merge it right away since this already has SIG approval. Otherwise, we can also make the change and attribute you in the PR as the originator of the fix.

Happy to do it either way, let us know.

**Comment by nilsandrey** [comment]
oh, I'll be busy for the weekend, just edit it yourself is ok thanks.

**Pull request closed without merging**